src/rocker/build/copy_test.go

```diff
@@ -380,6 +380,27 @@ func TestCopy_ListFiles_Excludes_FileInAnyDir(t *testing.T) {
 	}
 }
 
+func TestCopy_ListFiles_SymLink(t *testing.T) {
+	tmpDir, err := ioutil.TempDir("", "rocker-copy-test")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.RemoveAll(tmpDir)
+
+	includes := []string{
+		".",
+	}
+
+	link := tmpDir + "/link"
+	if err := os.Symlink("./", link); err != nil {
```

Comment:
I would add a few more files/links to make sure they are also copied, not just "./" dir.